test/cmd/e2e_helper/main.go

```diff
@@ -151,13 +151,23 @@ func (s *server) networkPingTest(w http.ResponseWriter, r *http.Request) {
 	}
 
 	c := http.Client{
-		Timeout: 10 * time.Second,
+		Timeout: 2 * time.Second,
```

Comment:
2 seconds is enough, and can return the timeout error quickly so that the test can run next retry.